Sources/Plugins/OsmEditing/OAOsmEditingPlugin.mm

```diff
@@ -110,41 +121,46 @@ - (void) updateLayers
 - (id<OAOpenStreetMapUtilsProtocol>)getPoiModificationUtil
 {
     if ([Reachability reachabilityForInternetConnection].currentReachabilityStatus != NotReachable && !_settings.offlineEditing)
-        return _remoteOsmUtil;
+        return _remoteUtil;
     else
-        return _localOsmUtil;
+        return _localUtil;
+}
+
+- (id<OAOpenStreetMapUtilsProtocol>)getPoiModificationLocalUtil
+{
+    return _localUtil;
 }
 
-- (id<OAOpenStreetMapUtilsProtocol>)getOfflineModificationUtil
+- (id<OAOpenStreetMapUtilsProtocol>)getPoiModificationRemoteUtil
 {
-    return _localOsmUtil;
+    return _remoteUtil;
 }
 
-- (id<OAOpenStreetMapUtilsProtocol>)getOnlineModificationUtil
+-(id<OAOsmBugsUtilsProtocol>)getOsmNotesRemoteUtil
 {
-    return _remoteOsmUtil;
+    return _remoteNotesUtil;
 }
 
--(id<OAOsmBugsUtilsProtocol>)getLocalOsmNotesUtil
+-(id<OAOsmBugsUtilsProtocol>)getOsmNotesLocalUtil
 {
-    return _localBugsUtil;
+    return _localNotesUtil;
 }
 
--(id<OAOsmBugsUtilsProtocol>)getRemoteOsmNotesUtil
+- (OAOsmBugsDBHelper *) getDBBug
```

Comment:
There is absolutely no need for this method in iOS. Objective C's singletons are easy to use without the need to call such methods (unlike Java).